docs/sections/user_guide/yaml/components/global_equiv_resol.rst

```diff
@@ -0,0 +1,31 @@
+.. _global_equiv_resol_yaml:
+
+global_equiv_resol_yaml
+=======================
+
+Structured YAML to run the WRF preprocessing component ``global_equiv_resol_yaml`` is validated by JSON Schema and requires the ``global_equiv_resol_yaml:`` block, described below. If ``global_equiv_resol_yaml`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
```

Comment:
```suggestion
Structured YAML to run the WRF preprocessing component ``global_equiv_resol`` is validated by JSON Schema and requires the ``global_equiv_resol:`` block, described below. If ``global_equiv_resol`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
```
The same thing happened in @NaureenBharwaniNOAA 's `esg_grid` PR. Do we have an error elsewhere that is creating a copy/paste problem?

Reply:
I'm not sure. That's quite interesting that we did the same thing.

Reply:
That's on me. I was looking at your PR and other drivers to see why my links weren't working properly and must have copied this over trying to come up with a solution.